chapter04/GamblersProblem.py

```diff
@@ -20,9 +20,13 @@
 # optimal policy
 policy = np.zeros(GOAL + 1)
 
-# state value
+# reward Value
+rewardValue = np.zeros(GOAL + 1)
+rewardValue[GOAL] = 1.0
+
+# state Value
 stateValue = np.zeros(GOAL + 1)
-stateValue[GOAL] = 1.0
+stateValue2 = np.zeros(GOAL + 1)
```

Comment:
I mean to do asynchronous DP. I think there won't be significant difference between asynchronous and synchronous DP.